catalogs/catalog-lakehouse-iceberg/src/main/java/org/apache/gravitino/catalog/lakehouse/iceberg/IcebergCatalogOperations.java

```diff
@@ -513,6 +514,13 @@ public Table createTable(
                           .build())
               .toArray(IcebergColumn[]::new);
 
+      // Gravitino NONE distribution means the client side doesn't specify distribution, which is
```

Comment:
Actually this isn't accurate for me. The unspecified distribution and unpartitioned should be different. But it may change more code. The fix may better than legacy implement. So I approve this.